src/psyclone/domain/lfric/algorithm/lfric_alg.py

```diff
@@ -44,6 +44,7 @@
 from psyclone.dynamo0p3 import DynKern
 from psyclone.errors import InternalError
 from psyclone.parse.kernel import get_kernel_parse_tree, KernelTypeFactory
+from psyclone.parse.utils import ParseError
```

Comment:
I must admit I'm not sure about the LFRicAlg class. The only use of self here is to references other methods within the class apart from the one static variable, so I don't see the need for a class here at all. It's also not clear to me if this is the best way to construct a new LFRic Algorithm layer code in general but I must admit I am in the dark on this. I also don't have any suggestions so this is just a comment.

Reply:
I've gone this route because @hiker always complains if I/we expose multiple routines in a module. He prefers the encapsulation provided by putting things inside a class. I suspect he's right but will let you have the debate with him :-)

Reply:
It doesn't feel right to me. The solution for what he wants would be to have separate modules for each routine. I'll have to remember to bring it up at the next telco.